### PR TITLE
Move atoms-rendering window types to root

### PR DESCRIPTION
**Pull request by joecowton1**
## What are you changing?

- move window types for atoms-rendering to the window.d.ts in the root @types folder
- add conditionals for `window.google` to youtube atom

## Why?

- Having the `window` definitions in a tsx file was causing issues when upgrading to the latest nx packages in https://github.com/guardian/csnx/pull/523

### CODE REVIEW

File: libs/@guardian/atoms-rendering/src/YoutubeAtomPlayer.tsx [hunk 1]
```diff
@@ -19,7 +19,7 @@ import type { AdTargeting, VideoEventKey } from './types';
 import type { PlayerListenerName } from './YouTubePlayer';
 import { YouTubePlayer } from './YouTubePlayer';
 
-declare class ImaManager {
+export declare class ImaManager {
```
Comment (sndrs) [inline libs/@guardian/atoms-rendering/src/YoutubeAtomPlayer.tsx:22]: i think this should not use `declare`, to stop it being ambient itself

```suggestion
export class ImaManager {
```

File: @types/window.d.ts [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import type { TeamSubscription } from '../libs/@guardian/libs/src/logger/@types/logger';
 import type { Switches } from '../libs/@guardian/libs/src/switches/@types/Switches';
+import { ImaManager } from 'libs/@guardian/atoms-rendering/src/YoutubeAtomPlayer';
```
Comment (sndrs) [inline @types/window.d.ts:3]: this path is wrong, which makes me thing the `declare` below was still making it ambient (assuming nothing errored?)

```suggestion
import { ImaManager } from '../libs/@guardian/atoms-rendering/src/YoutubeAtomPlayer';
```

File: @types/window.d.ts [hunk 3]
```diff
@@ -38,5 +39,13 @@ declare global {
 				pageViewId: string;
 			};
 		};
+		/**
+		 * Here we want to type the google object that will be added to window.
+		 * Since the imported google namespace is a value rather than a type, use typeof.
+		 */
+		google: typeof google;
```
Comment (sndrs) [inline @types/window.d.ts:46]: where does the `google` type come from?

Comment (mxdvl) [reply @types/window.d.ts:46]: (used to come from https://github.com/guardian/atoms-rendering/blob/6b4a78543898acf5fde44203b2b5a6ef693daec0/src/ima.d.ts)

File: libs/@guardian/atoms-rendering/src/YoutubeAtomPlayer.tsx (hunk 1 above)
Comment (joecowton1) [reply libs/@guardian/atoms-rendering/src/YoutubeAtomPlayer.tsx:22]: as in leave it as it was?

Comment (joecowton1) [reply libs/@guardian/atoms-rendering/src/YoutubeAtomPlayer.tsx:22]: ah ok great!

**Review by arelra — APPROVED** [review]
lgtm 👍

**Pull request merged**